src/Exceptionless.Core/Extensions/IdentityUtils.cs

```diff
@@ -106,7 +110,16 @@ public static class IdentityUtils {
         }
 
         public static string GetUserId(this IPrincipal principal) {
-            return IsTokenAuthType(principal) ? GetClaimValue(principal, UserIdClaim) : GetClaimValue(principal, ClaimTypes.NameIdentifier);
+            return IsUserAuthType(principal) ? GetClaimValue(principal, ClaimTypes.NameIdentifier) : null;
```

Comment:
This is a nice find, by returning null here, this shouldn't be an issue... It was used for web sockets for on connect and disconnect and you have to be user authed for those.